code/game/machinery/computer/apc_control.dm

```diff
@@ -6,16 +6,17 @@
 	req_access = list(ACCESS_CE)
 	circuit = /obj/item/circuitboard/computer/apc_control
 	light_color = LIGHT_COLOR_DIM_YELLOW
-	var/obj/machinery/power/apc/active_apc //The APC we're using right now
+	///The APC we're remotely connected to right now
+	var/obj/machinery/power/apc/active_apc
```

Comment:
bit unrelated to ur fix but do we clean up after this ref anywhere?

Reply:
handled the apc being destroyed a bit better, hopefully

Reply:
nice job, should be enough